pkg/repository/chartmuseum.go

```diff
@@ -162,6 +175,12 @@ func (c *chartmuseum) Poll() {
 			c.logger.Error(err, "failed to patch repository status")
 		}
 	}
+	if updateRepo {
```

Comment:
@Abirdcfly if we update repo by watcher, do we still need `helm repo update` in `componentplan`?

Reply:
Because the operator is a concurrent asynchronous program, I recommend keeping the `helm repo update` in `componentplan`